automated/linux/ethernet/test-eth-iperf3.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+source "../test_format.sh"
+
+#Worker Host IP_ADDRESS!
+IP_ATE="192.168.168.73"
+
+ip a show eth0 | grep inet
+
+echo "$TXT_Section1"
+echo "Connecting with ATE IP-Address: $IP_ATE"
+echo "$TXT_Section1"
+echo "$TXT_Section2"
+echo "Start: 1/2"
+echo "$TXT_Section2"
+iperf3 -t 1800 -4 -c $IP_ATE -t 10
+if [ $? -eq 0 ]
```

Comment:
Is it possible to check more Info(e.g. speed) to garantiere the ethernet interface works well

Reply:
I will improve this test